fiat-roles/src/main/java/com/netflix/spinnaker/fiat/config/RedisConfig.java

```diff
@@ -62,6 +65,9 @@ private static JedisPool createPool(
       redisPoolConfig = new GenericObjectPoolConfig();
     }
 
-    return new JedisPool(redisPoolConfig, host, port, timeout, password, database, null);
+    return new InstrumentedJedisPool(
+        registry,
+        new JedisPool(redisPoolConfig, host, port, timeout, password, database, null),
+        "fiat");
```

Comment:
I made this generic (I should deprecate `InstrumentedJedisPool`): https://github.com/spinnaker/kork/blob/master/kork-telemetry/src/main/java/com/netflix/spinnaker/kork/telemetry/InstrumentedProxy.java

Usage: https://github.com/spinnaker/kork/blob/master/kork-telemetry/src/test/groovy/com/netflix/spinnaker/kork/telemetry/InstrumentedProxySpec.groovy

Reply:
pending spinnaker/kork#313

Reply:
actually scratch that, InstrumentedProxy doesn't work here (it would just instrument the pool methods not the pooled object itself)